src/components/form/HdRange.vue

```diff
@@ -0,0 +1,329 @@
+<template>
+  <div :class="fieldClasses" class="range field">
+    <input
+      type="range"
+      v-model.number="currentValue"
+      :id="name"
+      :name="name"
+      :required="required"
+      :disabled="disabled"
+      :min="minValue"
+      :max="maxValue"
+      :step="rangeStep"
+      @focus="focusHandler"
+      @blur="blurHandler"
+    />
+    <div class="range__decoration" ref="decoration">
+      <div class="range__progress" ref="progress" />
+    </div>
+    <ul v-if="displayStepBullets" class="range__steps">
+      <li v-for="(steps, i) in stepsAmount" :key="i" />
```

Comment:
This is the "problematic" part.

Reply:
I fixed it, giving the number of steps as integer division and give a relative padding-right to the unsorted list. The value of the padding-right is the proportion of module of the division on the range "size".